google/cloud/compute/v1/compute.config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "converterVersion": "UNSPECIFIED",
+  "apiVersion": "",
+  "discoveryRevision": "",
+  "inlineSchemas" : [{
+    "schema": "",
+    "locations": {
+      "ErrorSet": [
```

Comment:
Please confirm this is a reasonable set for this inline schema that was causing the original problem.